src/service/data_products/genome_attributes.py

```diff
@@ -266,27 +265,12 @@ async def get_genome_attributes_meta(
     ) -> col_models.ColumnarAttributesMeta:
     storage = app_state.get_app_state(r).arangostorage
     _, load_ver = await get_load_version(storage, collection_id, ID, load_ver_override, user)
-    meta = await _get_genome_attributes_meta_internal(
-        storage, collection_id, load_ver, load_ver_override)
+    meta = await get_columnar_attribs_meta(
+        storage, names.COLL_GENOME_ATTRIBS_META, collection_id, load_ver, load_ver_override)
     meta.columns = [c for c in meta.columns if not c.non_visible]
     return meta
```

Comment:
This code looks identical to the code in samples other than the arango collection

Reply:
👍

Reply:
I was thinking only the meta endpoints would be updated here, the other endpoints could keep using the method as it was. E.g. just add a new method that calls the current method

Reply:
👍

Reply:
:+1: